plugins/catalog/src/components/CatalogPage/CatalogPage.tsx

```diff
@@ -137,12 +137,12 @@ const CatalogPage: FC<{}> = () => {
               <span role="img" aria-label="wave" style={{ fontSize: '125%' }}>
                 👋🏼
               </span>
-              Welcome to Backstage, we are happy to have you. Start by checking
-              out our
+              &nbsp;&nbsp;&nbsp;Welcome to Backstage, we are happy to have you.
```

Comment:
That sounds like you should have some margin or padding on the waving hand icon's right side instead